src/components/core/popoverMenu/popoverItem.vue

```diff
@@ -31,7 +31,17 @@
 		</a>
 		<!-- If item.input is set instead, an put will be used -->
 		<span v-else-if="item.input" class="menuitem">
-			<input :id="key" :type="item.input" :class="item.input"
+			<!-- does not show if input is checkbox -->
+			<span v-if="item.input != 'checkbox'" :class="item.icon" />
+			<!-- only shows if input is text -->
+			<form v-if="item.input == 'text'"
+				:class="item.input" @submit.prevent="item.action">
+				<input :type="item.input" :value="item.value" :placeholder="item.placeholder">
+				<input v-if="item.input == 'text'" type="submit" value=""
+					class="icon-confirm">
+			</form>
+			<input v-else
```

Comment:
Put the first on the same line please ;)

Reply:
? sorry the first? I think I can remove this v-if actually is that your point, or the class I think the linted wanted it on a separate line

Reply:
First attribute, sorry :)

Reply:
Arh you mean for `<input v-else` got you!